src/aliases/api/api_test.go

```diff
@@ -0,0 +1,397 @@
+package aliasapi_test
```

Comment:
Following what we did in `/stores/api` unit test are being written over handlers file so we test one by one each endpoint and each edge case. Having them all together in a single file it makes things harder to maintain and follow

Reply:
done